### PR TITLE
EDITOR-#22 [Backend] Article Resolvers Completed

### LINKED ISSUE #22: [Backend] :sparkles: Article Resolver
model Article
- Query.tsx
   - [x] AllArticle
- Mutation.tsx
   - [x] CreateArticle
   - [x] DeleteArticle
   - [x] UpdateArticle
- Subscription.tsx
   - [x] ArticleCreated
   - [x] ArticleDeleted
   - [x] ArticleUpdated

### PR DESCRIPTION
**Pull request by Jackbear8868**
model Article
Query.tsx
- [x]  AllArticle

Mutation.tsx
- [x] CreateArticle
- [x]  DeleteArticle
- [x]  UpdateArticle

Subscription.tsx
- [x]  ArticleCreated
- [x]  ArticleDeleted
- [x]  ArticleUpdated

### CODE REVIEW

**Comment by Jackbear8868** [comment]
model ArticleComment
Query.tsx
- [x]  AllArticleComment
Mutation.tsx
- [x]  CreateArticleComment
- [x]  DeleteArticleComment
- [x]  UpdateArticleComment
Subscription.tsx
- [x]  ArticleCommentCreated
- [x]  ArticleCommentDeleted
- [x]  ArticleCommentUpdated

**Review by ChenBingWei1201 — CHANGES_REQUESTED** [review]
please fix some errors

**Review by ChenBingWei1201 — COMMENTED** [review]
- `zap`, `isMe`, and `bombFish` should keep their value when being updated.
- I forgot to add `zap` into `ArticleComment` and `QuestionComment` in `schema.prisma`
Would you please help me add it to them? Thanks a lot.

File: backend/src/resolvers/Mutation.ts [hunk 1]
```diff
@@ -152,6 +152,193 @@ const Mutation ={
...
+      where: {
+        id: id
+      },
+      data: {
+        writerId: writerId,
+        date: date,
+        title: title,
+        content: content,
+        tags: tags,
+        zap: 0,
```
Comment (ChenBingWei1201) [inline backend/src/resolvers/Mutation.ts]: `UpdateArticle` should keep the original zap numbers instead of resetting them.

**Review by ChenBingWei1201 — CHANGES_REQUESTED** [review]
- `zap`, `isMe`, and `bombFish` should keep their value when being updated.
- I forgot to add `zap` into `ArticleComment` and `QuestionComment` in `schema.prisma`
Would you please help me add it to them? Thanks a lot.

**Pull request merged**